docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,872 @@
+
+Influencers:
+
+* `user_agent.name`
+* `error.exception.message.keyword`
+* `error.page.url`
+* `service.name`
+
+Bucket span: 15m.
+
+Function: `high_non_zero_count`.
+
+
+decreased_throughput_jsbase::
+decreased_throughput_nodejs::
+
+* For APM data where `agent.name` is `js-base` or `nodejs`.
+* Models the transaction rate of the application.
+* Detects periods during which the application is processing fewer requests 
+than normal.
+
+Influencers:
+
+* `service.name`
+* `transaction.name`
+
+Bucket span: 15m.
+
+Function: `low_count`.
+
+
+high_count_by_user_agent_jsbase::
+
+* For APM data where `agent.name` is `js-base`.
+* Models the request rate of user agents (`partition_field_name` is 
+  `user_agent.name`).
+* Detects user agents that are making requests at a suspiciously high rate.
+
+This job is useful in identifying bots.
+
+Influencers:
+
+* `service.name`
+* `user_agent.name`
+
+Bucket span: 15m.
+
+Function: `high_non_zero_count`.
+
+
+high_mean_response_time::
+
+* For transaction data where `processor.event` is `transaction` and 
+`transaction.type` is `request`.
+* Models response time duration of transactions.
+* Detects anomalies in high mean of transaction duration.
+
+Bucket span: 15m.
+
+Function: `high_mean`.
+
+
+[float]
+[[ootb-ml-jobs-auditbeat]]
+=== Auditbeat
+
+These {anomaly-job} wizards appear in {kib} if you use 
+{auditbeat-ref}/index.html[{auditbeat}] to audit process activity on your 
+systems.
+
+docker_high_count_process_events_ecs::
+
+* For Auditbeat data where `event.module` is `auditd` and `container.runtime` is 
+`docker`.
+* Models process execution rates (`partition_field_name` is `container.name`).
+* Detects unusual increases in process execution rates in Docker containers.
+
+Influencers:
+
+* `container.name`
+* `process.executable`
+
+Bucket span: 1h.
+
+Function: `high_count`.
+
+
+docker_rare_process_activity_ecs::
+
+* For Auditbeat data where `event.module` is `auditd` and `container.runtime` is 
+`docker`.
+* Models occurrences of process execution (`partition_field_name` is 
+  `container.name`).
+* Detects rare process executions in Docker containers.
+
+Influencers:
+
+* `container.name`
+* `process.executable`
+
+Bucket span: 1h.
+
+Function: `rare`.
+
+
+hosts_high_count_process_events_ecs::
+
+* For Auditbeat data where `event.module` is `auditd`.
+* Models process execution rates (`partition_field_name` is `host.name`).
+* Detects unusual increases in process execution rates.
+
+Influencers:
+
+* `host.name`
+* `process.executable`
+
+Bucket span: 1h.
+
+Function: `high_non_zero_count`.
+
+
+hosts_rare_process_activity_ecs::
+
+* For Auditbeat data where `event.module` is `auditd`.
+* Models process execution rates (`partition_field_name` is `host.name`).
+* Detects rare process executions on hosts.
+
+Influencers:
+
+* `host.name`
+* `process.executable`
+
+Bucket span: 1h.
+
+Function: `rare`.
+
+
+[float]
+[[ootb-ml-jobs-logs-ui]]
+=== Logs UI
+
```

Comment:
Do these jobs appear by default in the Logs UI? If so, I think we should clarify how you get them with something like this:
```suggestion
tag::logs-jobs[]
These {anomaly-jobs} appear by default in the {kibana-ref}/xpack-logs.html[Logs app] in {kib}. 
```